### ISSUE TITLE
Console not working

### ISSUE THREAD

**Issue by ezhkov** [CompuIves/codesandbox-client#1156]
# 🐛 bug report

## Description of the problem
Integrated console not working

### Your Environment
| Software         | Name/Version|
| ---------------- | ---------- |
| Сodesandbox  | 
| Browser | Chrome
| Operating System | MacOS Mojave

**Comment by js-mode** [comment]
I am having the exact same issue and I am on Firefox (MacOS Sierra) - (example sandbox: https://codesandbox.io/s/5xmrmvl7xn)

In the browser console there are some errors that are showing up (see screenshot below).  Also there is no terminal tab next to the console like there used to be. 

I have tried creating a VanillaJS & React code sandbox and I can get the DOM to update with new code, but nothing ever in the console.   

Previously if I tried to run npm start I would get a syntax error, so I refreshed my GitHub OAuth to see if that fixed it -- but no luck. 

<img width="1733" alt="screen shot 2018-10-03 at 6 44 36 pm" src="https://user-images.githubusercontent.com/29901608/46448639-ba28c780-c73c-11e8-8bcf-5e1e8ed2020d.png">
